.github/workflows/build.yml

```diff
@@ -15,32 +15,32 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Setup Java
+      - name: Setup java (tests)
         uses: actions/setup-java@v4
         with:
-          java-version: 21
           distribution: 'oracle'
+          java-version: '22'
           cache: 'maven'
-          server-id: ossrh
-          server-username: MAVEN_USERNAME
-          server-password: MAVEN_CENTRAL_TOKEN
 
-      - name: Build project (macOS)
-        if: runner.os == 'macOS'
+      - name: Build project (Linux/macOS)
+        if: runner.os != 'Windows'
         run: |
-          ./mvnw -B -ntp verify -f rta
+          ./mvnw -B -ntp -Ptest verify -f rta
 
       - name: Build project (Windows)
         if: runner.os == 'Windows'
         run: |
-          .\mvnw -B -ntp verify -f rta
+          .\mvnw -B -ntp -Ptest verify -f rta
 
-      - name: Build project (Linux)
-        if: runner.os == 'Linux'
-        run: |
-          export DISPLAY=:90
-          Xvfb -ac :90 -screen 0 1280x1024x24 > /dev/null 2>&1 &
-          ./mvnw -B -ntp verify -f rta
+      - name: Setup Java (distribution)
+        uses: actions/setup-java@v4
+        with:
+          java-version: 21
```

Comment:
can't we build on 22 and target 21?

Reply:
I'm just keeping here what we had for distribution. For tests, JDK 22+ is mandatory because of JavaFX 24-ea. For publishing, we use JavaFX 23.0.1, that works with 21, but we could use 22 of course.

Reply:
But I see what you mean. We can avoid needing two java-setups, one with JDK 22 will be enough. I've updated the workflows.